test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -421,6 +421,148 @@ describe('the rubicon adapter', () => {
 
           delete window.DigiTrust;
         });
+
+        describe('digiTrustId config', () => {
+          var origGetConfig;
+          beforeEach(() => {
+            window.DigiTrust = {
+              getUser: sinon.spy()
+            };
+            origGetConfig = window.pbjs.getConfig;
+          });
+
+          afterEach(() => {
+            delete window.DigiTrust;
+            window.pbjs.getConfig = origGetConfig;
```

Comment:
I was gonna merge this, cause it has two LGTMs, but... use $$PREBID_GLOBAL$$ rather than pbjs, even in unit tests.

See #1437 and #1462